sklearn/metrics/_plot/tests/test_plot_confusion_matrix.py

```diff
@@ -264,3 +263,37 @@ def test_confusion_matrix_text_format(pyplot, data, y_pred, n_classes,
     text_text = np.array([
         t.get_text() for t in disp.text_.ravel()])
     assert_array_equal(expected_text, text_text)
+
+
+@pytest.mark.parametrize("values_format", [None])
+def test_confusion_matrix_standard_format(pyplot, data, y_pred, n_classes,
+                                          fitted_clf, values_format):
+    X, y = data
+    cm = np.array([[10000000, 0], [29, 123123]])
+    disp = plot_confusion_matrix(fitted_clf, X, y,
+                                 include_values=True,
+                                 values_format=values_format)
+    disp.confusion_matrix = cm
+    plotted_text_values = disp.plot().text_
+    # Values should be shown as whole numbers 'd',
+    # except the first number which should be shown as 1e+07
+    test = []
+    for i in plotted_text_values.ravel():
+        test.append(i.get_text())
+    test = np.asarray(test)
```

Comment:
Avoid making these into arrays. Just use a bare `assert` statement to compare the two lists, please

Reply:
✔